style.css

```diff
@@ -32,29 +28,107 @@ body {
   font-size: 18px;
   line-height: 24px;
   color: #fff;
+  padding: 2%;
+}
+
+@media screen and (min-width: 375px) and (max-width: 768px) {
+  .menu-items.show {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start space-around;
+    align-items: start;
+    background-color: #3c3a39;
+    height: 100%;
+    width: 100%;
+    transition: 0.3s;
+    position: fixed;
+    top: 0;
+    left: 0;
+    list-style: none;
+    margin: 0;
+    padding-top: 15%;
+    padding-right: 10%;
+  }
+
+  .menu-items.show hr {
+    min-width: 279px;
+    height: 1px;
```

Comment:
- You have done great so far. :+1: :+1: But there are some changes you need to make for moving to the next milestone. :+1: 
- [ ] Looking at the video below, you can see that when I open the mobile menu and scroll, the Project section content tends to go over the menu. this is not great for the user experience, I'll suggest you fix this. You can fix this by increasing the z-index` property of the mobile menu. :+1: :+1:  

- - Video for reference:

https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/107632780/e3db946c-fdf1-4681-96b8-5ab150996742